App.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import Login from './Login';
```

Comment:
Consider using lazy loading for the Login and Dashboard components to improve performance. You can use React.lazy() for this purpose.